interface/wx/cpp.h

```diff
@@ -52,9 +52,20 @@
 
     Please use the standard macro instead.
 
+    @see __WXFUNCTION_SIG__
+
     @header{wx/cpp.h}
 */
 #define __WXFUNCTION__ __func__
 
+/**
+    Display the current function's full signature, if available.
+
+    Falls back to the function name (i.e., @c \__func__) if not available.
```

Comment:
Would be nice to show a simple example showing the difference.

Reply:
Done. Used a member function with a `const` decorator to really show off the difference.